### ISSUE TITLE
Connect to remote sensors

### ISSUE THREAD

**Issue by Zren** [Zren/plasma-applet-sysmonitordash#23]
![2019-12-05___17-16-19](https://user-images.githubusercontent.com/416367/70278813-f9103e00-1782-11ea-8be2-b34bf5ee0d8e.png)

KSysGuard can monitor remote KDE installations.
https://docs.kde.org/trunk5/en/kde-workspace/ksysguard/ksysguard.pdf
2.3.7    Connecting to other hosts

* File > New Tab
* Select the new tab/sheet.
* File > Monitor Remote Machine

I already have ssh setup using `ssh -t username@192.168.1.10`
https://askubuntu.com/questions/46930/how-can-i-set-up-password-less-ssh-login

So I used `username@192.168.1.10` in the "Monitor Remote Machine" popup and it worked.

You can then drag the "CPU Load/CPU 1/Total Load" onto a graph.

I don't think you can use a remote sensor in my widget though.
`~/.config/ksysguardrc` has:
SelectedSheets[$e]=ProcessTable.sgrd,SystemLoad2.sgrd,Sheet 1.sgrd,Sheet 3.sgrd,Sheet 4.sgrd

Using `locate .sgrd`
Finds: `~/.local/share/ksysguard/Sheet 4.sgrd`

Which distinctly uses:
```
<beam sensorName="cpu/cpu0/TotalLoad" hostName="username@host" color="0xff0057ae" sensorType="float"/>
```

Which I tried modifying to use regexpSensorName like the `SystemLoad2.sgrd`:
```
<beam regexpSensorName="cpu/cpu.*/TotalLoad" hostName="username@192.168.1.10" color="0xffff8000,0xffe20800,0xfff3c300,0xffff0000,0xff00a900,0xff0028a9" sensorType="float"/>
```

**Comment by Zren** [comment]
https://github.com/KDE/plasma-workspace/blob/master/dataengines/systemmonitor/systemmonitor.cpp

The systemmonitor dataengine appears to hardcode `localhost` so it doesn't appear the widget can monitor remote sensors.

**Comment by N7DR** [comment]
> KSysGuard can monitor remote KDE installations.
> https://docs.kde.org/trunk5/en/kde-workspace/ksysguard/ksysguard.pdf

Yes, ksysguard is what I use in Trinity(KDE3). But I can't see how to put ksysguard in a panel in KDE5 :-( That inability is what led me to look for a different widget, which is when I found yours.

Annoying about the systemmonitor dataengine not allowing for interrogation of remote systems
